packages/formation-react/src/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -39,16 +28,26 @@ class Breadcrumbs extends React.Component {
   };
 
   render() {
-    const { ariaLabel, mobileFirstProp } = this.props;
-    const breadcrumbId = this.props.id || uniqueId('va-breadcrumbs-');
-    const breadcrumbListId =
-      this.props.listId || uniqueId('va-breadcrumbs-list-');
+    const { ariaLabel, customClasses, className, id, listId, mobileFirstProp } = this.props;
+
+    // Derive IDs.
+    const breadcrumbId = id || uniqueId('va-breadcrumbs-');
+    const breadcrumbListId = listId || uniqueId('va-breadcrumbs-list-');
+
+    // Warn if they are using legacy props.
+    if (customClasses) {
+      console.warn('The prop `customClasses` for the <Breadcrumbs /> component has been deprecated. Please use the prop `className` instead.')
+    }
```

Comment:
I think a warning would be nice if it were a heads-up about removing support in an upcoming release, but since we're bumping the minor version I don't think this is really necessary. Up to you though if you feel strongly.

Reply:
That makes sense! There was just [this one place in `vets-website`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/components/Breadcrumbs.jsx#L7) where that prop is still used, so as long as the PR to bump formation-react on vets-website also includes a fix for that we don't need the warning imo 🙂